CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKIconsRenderable.java

```diff
@@ -1195,96 +1177,76 @@ public int RecordDisplayCommandBuffer(VkCommandBufferInheritanceInfo inheritance
         CVKAssertNotNull(cvkSwapChain);
                 
         int ret;     
-        try (MemoryStack stack = stackPush()) {
- 
-            CVKCommandBuffer commandBuffer = commandBuffers.get(imageIndex);
-            CVKAssert(commandBuffer != null);
-            CVKAssert(displayPipelines.get(imageIndex) != null);
-            
-            commandBuffer.BeginRecordSecondary(VK_COMMAND_BUFFER_USAGE_RENDER_PASS_CONTINUE_BIT,
-                                                           inheritanceInfo);
-            
-            // Set the dynamic viewport and scissor
-            commandBuffer.viewPortCmd(cvkSwapChain.GetWidth(), cvkSwapChain.GetHeight(), stack);
-            commandBuffer.scissorCmd(cvkVisualProcessor.GetCanvas().GetCurrentSurfaceExtent(), stack);
-            
-            // Bind the graphics pipeline
-            commandBuffer.BindGraphicsPipelineCmd(displayPipelines.get(imageIndex));
-
-            LongBuffer pVertexBuffers = stack.longs(vertexBuffers.get(imageIndex).GetBufferHandle());
-            LongBuffer offsets = stack.longs(0);
-
-            // Bind verts
-            vkCmdBindVertexBuffers(commandBuffer.GetVKCommandBuffer(), 0, pVertexBuffers, offsets);
-
-            // Bind descriptors
-            vkCmdBindDescriptorSets(commandBuffer.GetVKCommandBuffer(), 
-                                    VK_PIPELINE_BIND_POINT_GRAPHICS,
-                                    hPipelineLayout, 
-                                    0, 
-                                    stack.longs(pDescriptorSets.get(imageIndex)), 
-                                    null);            
-            
-            vkCmdDraw(commandBuffer.GetVKCommandBuffer(),
-                      GetVertexCount(),  //number of verts == number of digits
-                      1,  //no instancing, but we must draw at least 1 point
-                      0,  //first vert index
-                      0); //first instance index (N/A)
-            
-            ret = vkEndCommandBuffer(commandBuffer.GetVKCommandBuffer());
-            checkVKret(ret);
-        }
+         
+        CVKCommandBuffer commandBuffer = commandBuffers.get(imageIndex);
+        CVKAssert(commandBuffer != null);
+        CVKAssert(displayPipelines.get(imageIndex) != null);
+
+        commandBuffer.BeginRecordSecondary(VK_COMMAND_BUFFER_USAGE_RENDER_PASS_CONTINUE_BIT, inheritanceInfo);
+
+        commandBuffer.SetViewPort(cvkSwapChain.GetWidth(), cvkSwapChain.GetHeight());
+        commandBuffer.SetScissor(cvkVisualProcessor.GetCanvas().GetCurrentSurfaceExtent());
+
+        commandBuffer.BindGraphicsPipeline(displayPipelines.get(imageIndex));
+        commandBuffer.BindVertexInput(vertexBuffers.get(imageIndex).GetBufferHandle());
+
+        // Push MVP matrix to the shader
```

Comment:
It's not the MVP matrix, it's the mvmatrix (technically I think view or camera matrix is more accurate).

Reply:
Fixed comment